build_scripts/dependencies_static_analysis_test_cov.py

```diff
@@ -0,0 +1,129 @@
+"""Koninklijke Philips N.V., 2019 - 2020. All rights reserved.
+build script to building the similarity tool"""
+import platform
+
+from install_dependencies import install_pip
+from subprocess_calls import call_subprocess
+
+
+def install_aspell():
+    """
+    Installs Aspell and configure for english
+    """
+    if str(platform.system()).upper() == "LINUX":
+        call_subprocess("sudo apt-get update -qq")
+        call_subprocess("sudo apt-get install --assume-yes aspell aspell-en")
+        print("Stage Install aspell -- COMPLETED & PASSED --")
+    else:
+        print("Please install and configure Aspell for english")
+
+
+def install_npm_ctags_packages():
+    """
+    Installs jscpd and configure for english
+    """
+    if str(platform.system()).upper() == "LINUX":
+        call_subprocess("sudo npm install")
+        call_subprocess("sudo npm install -g jscpd@3.2.1")
+        call_subprocess("sudo npm i -g yaml-lint@1.2.4")
+        call_subprocess("sudo npm i -g markdownlint-cli@0.23.1")
+        call_subprocess("sudo apt-get install ctags")
+        call_subprocess('export PATH="$PATH:/usr/bin/ctags"')
+        print("Stage Install jscpd, ctags, markdownlint & ymllint -- COMPLETED & PASSED --")
+    else:
+        print("Please install and configure jscpd, ctags, markdownlint & ymllint")
```

Comment:
would you consider creating a new function for ctag alone and call it in main

Reply:
Corrected